src/ec/packages/ec-component-accordion2/accordion2.story.js

```diff
@@ -1,36 +1,51 @@
 /* eslint-disable no-param-reassign */
 import { storiesOf } from '@storybook/html';
-import { withKnobs, text } from '@storybook/addon-knobs';
+import { withKnobs, text, select } from '@storybook/addon-knobs';
 import { withNotes } from '@ecl-twig/storybook-addon-notes';
+import { buttonLabels, getExtraKnobs } from '@ecl-twig/story-utils';
 import withCode from '@ecl-twig/storybook-addon-code';
-
 import defaultSprite from '@ecl/ec-resources-icons/dist/sprites/icons.svg';
 import demoData from './demo/data';
 
 import accordion2 from './ecl-accordion2.html.twig';
 import notes from './README.md';
 
+const prepareAccordion2 = data => {
+  data.items.forEach((item, index) => {
+    const { id, level, toggle, content } = item;
+
+    item.id = select(`item[${index}].id`, [id], id, buttonLabels.required);
+    item.level = select(
+      `item[${index}].level`,
```

Comment:
@Joosthe..can you take a look at the demo data? Item doesn't exist, i wrote you this already.
This is:
`items[${index}].level`

and the same applies to the other knobs. 

**But this should be editable, the level is the heading that will be generated, this should be a select, yes, but with six values (1, 2, 3, 4, 5, 6) with the default set to 3 (as it is in the template), at least basing on the template we have.**

Reply:
oke i've made an array for this.